README.md

```diff
@@ -63,16 +66,16 @@ $config->setMetadataRefreshIntervalMs(10000);
 $config->setMetadataBrokerList('10.13.4.159:9192');
 $config->setBrokerVersion('0.9.0.1');
 $config->setRequiredAck(1);
-$config->setIsAsyn(false);
+$config->setIsAsyn(true);
```

Comment:
I didn't manage to make this flag behave the way I understood it should, so I'd suggest to keep the way it was

Reply:
reverting to false then or should I remove it completely?

Reply:
let's revert for now 😄